system_test/aest_sync_SUITE.erl

```diff
@@ -625,18 +627,18 @@ net_split_mining_power(Cfg) ->
     SyncLength = 20,
     ExtraLength = 3,
 
-    Net1Nodes = [net1_node1, net1_node2],
-    Net2Nodes = [net2_node1, net2_node2, net2_node3, net2_node4],
+    Net1Nodes = [net1_node1],
+    Net2Nodes = [net2_node1, net2_node2, net2_node3],
     AllNodes = Net1Nodes ++ Net2Nodes,
 
-    setup_nodes([?NET1_NODE1, ?NET1_NODE2, ?NET2_NODE1,
-                 ?NET2_NODE2, ?NET2_NODE3, ?NET2_NODE4], Cfg),
+    setup_nodes([?NET1_NODE1, ?NET1_NODE2,
```

Comment:
Why net1_node2 setup but not started? Leftover?

Reply:
It's not a left-over, the network is made for 6 nodes and they try to find each others addresses. It would crash hard if the node looking for is not configured, but for not starting it the penalty is low.

In this way we can easily increase the nr of nodes if one has a fast machine and needs to experiment a bit.